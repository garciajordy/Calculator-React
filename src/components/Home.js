import React from 'react';

const Home = () => (
  <div className="home">
    <h3>Welcome to our page!</h3>
    <p>
      This is the number one math magicians website in the world with loads of fun quotes and apps!
    </p>
    <p>
      Don&apos;t forget to checkout the Quote page and the Calculator page
    </p>
  </div>
);

export default Home;
