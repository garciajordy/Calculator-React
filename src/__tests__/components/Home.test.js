import React from 'react';
import { render, cleanup, screen } from '@testing-library/react';
import Home from '../../components/Home';

afterEach(cleanup);

it('should take a snapshot', () => {
  const { asFragment } = render(<Home />);
  expect(asFragment(<Home />)).toMatchSnapshot();
});
it('Should display header information', () => {
  render(<Home />);
  expect(screen.getByText('Welcome to our page!')).toBeInTheDocument();
  expect(screen.getByText('This is the number one math magicians website in the world with loads of fun quotes and apps!')).toBeInTheDocument();
});
