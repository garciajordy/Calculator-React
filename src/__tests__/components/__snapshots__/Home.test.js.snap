// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="home"
  >
    <h3>
      Welcome to our page!
    </h3>
    <p>
      This is the number one math magicians website in the world with loads of fun quotes and apps!
    </p>
    <p>
      Don't forget to checkout the Quote page and the Calculator page
    </p>
  </div>
</DocumentFragment>
`;
